### ISSUE TITLE
NullPointerException When driver is closed

### ISSUE THREAD

**Issue by bebef1987** [applitools/eyes.sdk.java3#92]
If the driver is closed and can't provide the session ID then we will get a NullPointer exception

https://github.com/applitools/eyes.sdk.java3/blob/f4388d71b855864423bf91e1a59c0ae3e01d0a6e/eyes.selenium.java/src/main/java/com/applitools/eyes/selenium/Eyes.java#L2296

**Comment by bebef1987** [comment]
I'm integrating Applitools with Serenity.

After the test is finished(driver is closed) we call a event handler to close eyes.
At this point we get the Null Pointer exception

As a regression range for this issue we have V3.32(Working fine) -> V3.33(Null Pointer exception)
```

java.lang.NullPointerException
	at com.applitools.eyes.selenium.Eyes.getAUTSessionId(Eyes.java:2294)
	at com.applitools.eyes.EyesBase.close(EyesBase.java:771)
	at com.garmin.testing.helpers.applitools.AppliToolsHelper.testFinished(AppliToolsHelper.java:185)
	at com.garmin.testing.helpers.applitools.ApplitoolsStepsListener.testFinished(ApplitoolsStepsListener.java:78)
	at net.thucydides.core.steps.StepEventBus.testFinished(StepEventBus.java:295)
	at net.thucydides.junit.listeners.JUnitStepListener.testFinished(JUnitStepListener.java:96)
```

**Comment by itaibh** [comment]
You just stated the problem - you close Eyes *AFTER* you close the driver. Just close Eyes first. The driver should be the last thing you close.

**Comment by bebef1987** [comment]
I know that. 
But...
Serenity manages the driver. We just access it and use it when we need it.
Because of this we can't control when it gets's closed.

**Comment by itaibh** [comment]
OK, I'll add a task for it.

**Comment by bebef1987** [comment]
Any updates on this??
We are still struggling with NullPointer exceptions

**Comment by danielputerman** [comment]
Hi, yes, a fix is available in version 3.37.1 . 
Let me know if you still have any issues.
